Scripts/build-nightly.cmd

```diff
@@ -39,7 +40,7 @@ for /f "tokens=* usebackq" %%A in (`tzutil /g`) do (
 @echo
 set targets=Build
 if not "%~1" == "" set targets=%~1
-"%msbuildpath%\msbuild.exe" -t:%targets% nightly.proj /fl /flp:logfile=build.log 
+"%msbuildpath%\msbuild.exe" -t:%targets% nightly.proj /fl /flp:logfile=build.log /bl
```

Comment:
Should the `/bl` flag also be in the `build-canary` & `build-stable` scripts?

Reply:
Can be done of course.
Objective was to start with Nightly.

Reply:
@Wagnerp 
On second thought it wont hurt putting it in there straight away.
So imo, let's put the switch in those two as well...

Reply:
@Wagnerp 
Added both.